tests/migrator/test_books.py

```diff
@@ -490,7 +490,7 @@ def test_urls(app):
         )
         check_transformation(
             """
-            <datafield tag="8564" ind1=" " ind2=" ">
+            <datafield tag="856" ind1="4" ind2=" ">
```

Comment:
did you check for occurrences in the data of `856__` to be sure we can replace the ind1 for this test?

Reply:
I didn't replace it, before it was `tag="8564"` and I moved the 4 to the ind1.